doc-files/release-notes-16.md

```diff
@@ -0,0 +1,115 @@
+# Release Notes for JavaFX 16
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 16 release. JavaFX 16 requires JDK 11 or later.
+
+## List of Fixed Bugs
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8253597](https://bugs.openjdk.java.net/browse/JDK-8253597)|TreeTableView: must select leaf row on click into indentation region|controls
+[JDK-8253696](https://bugs.openjdk.java.net/browse/JDK-8253696)|WebEngine refuses to load local "file:///" CSS stylesheets when using JDK 15|web
+[JDK-8248490](https://bugs.openjdk.java.net/browse/JDK-8248490)|[macOS] Undecorated stage does not minimize|window-toolkit
+[JDK-8252446](https://bugs.openjdk.java.net/browse/JDK-8252446)|Screen.getScreens() is empty sometimes|graphics
+[JDK-8251941](https://bugs.openjdk.java.net/browse/JDK-8251941)|ListCell: visual artifact when items contain null values|controls
+[JDK-8176270](https://bugs.openjdk.java.net/browse/JDK-8176270)|Adding ChangeListener to TextField.selectedTextProperty causes StringOutOfBoundsException|controls
+[JDK-8255415](https://bugs.openjdk.java.net/browse/JDK-8255415)|Nested calls to snap methods in Region give different results|graphics
+[JDK-8246343](https://bugs.openjdk.java.net/browse/JDK-8246343)|Fix mistakes in FX API docs|other
```

Comment:
This one is from an older version.

Reply:
All original issues are added to the Release Notes if an issue is a backport issue. It seems like there is backport issue against JDK-8246343 with fix version `openjfx16`.

Reply:
There are some "backport" issues that are sort of forward ported but still have type backport. For example, issues fixed in 15.0.1 are "backported" to 16.

Reply:
There are issues in 15 with a backport to 16. I'd be surprised if that were the case for 15.0.1 (and if so, they would still be in the release notes for 16 if they didn't ship with 15 GA).

https://bugs.openjdk.java.net/browse/JDK-8246343 is the case of a bug that was fixed in 15 and then synced into mainline for 16. So it should be ignored.

Reply:
Right, I was confused when I saw the tag 15.0.1+0 but it is indeed in 15.

Reply:
There might be other issues like that. Maybe the backport issue type should be filtered.

Reply:
This has already been taken care of. I have removed all the issues which were already shipped with 15GA and then backported to 16.